code/modules/mentor/verbs/mentorhelp.dm

```diff
@@ -0,0 +1,106 @@
+/client/verb/mentorhelp(msg as text)
+	set category = "Mentor"
+	set name = "Mentorhelp"
+
+	//clean the input msg
+	if(!msg)
+		return
+	msg = sanitize(copytext(msg,1,MAX_MESSAGE_LEN))
+	if(!msg || !mob)
+		return
+	if(src.prefs.muted & MUTE_MENTORHELP)
+		to_chat(src, "<font color='red'>You are unable to use mentorhelp (muted).</font>")
+		return
+	var/show_char = CONFIG_GET(flag/mentors_mobname_only)
+	var/mentor_msg = "<span class='mentornotice'><b><font color='#3280ff'>MENTORHELP:</b> <b>[key_name_mentor(src, 1, 0, 1, show_char)]</b>:</font> [msg]</span>"
+	log_mentor("MENTORHELP: [key_name_mentor(src, 0, 0, 0, 0)]: [msg]")
+
+	for(var/client/X in GLOB.mentors)
+		X << 'sound/items/bikehorn.ogg'
```

Comment:
Maybe we should add a sound specific to Mentorhelp?

Reply:
We can change admin bwoink to _adminhelp and give mentors adminhelp in the .ogg sound files
/s

Reply:
I think something other than the bikehorn would be better. Maybe something like the announcement sound?